src/app/logic/player.js

```diff
@@ -0,0 +1,44 @@
+export class Player {
+  constructor() {
+    this.correctAnswersScore = 0;
+    this.amountOfAnsweredQuestions = 0;
+    this.detailQuestionData = [];
+    this.endGameData = {};
+  }
+
+  registerAnswer(correctAnswer, playerAnswer, isanswerCorrect) {
+    if (this.playerAnswer === false) {
```

Comment:
How about `!this.playerAnswer?`

Reply:
done! will be in new commit. I must get used to that shorter form

Reply:
update, it looks that will be simple to change, but make a lot of problems